### PR TITLE
sys: crypto: Remove 3DES crypto

### PR DESCRIPTION
**Pull request by emmanuelsearch**
A few days ago at a [research seminar on RIOT](http://riot-os.org/seminar-2017.html), a cryptographer complained that we "still" support 3DES, which is considered legacy.
Googling around, this seems true.

Hence this PR, which removes 3DES support. 
Are there contrary opinions on this? 
I plan to also email devel@ and user@ to pre-warn of this pending removal.

### CODE REVIEW

**Comment by kaspar030** [comment]
+1

**Comment by kaspar030** [comment]
please squash

**Comment by emmanuelsearch** [comment]
cppcheck complains weirdly that:

```
$ ./dist/tools/externc/check.sh
file does not have a C++ compatible header: 'pkg/nordic_softdevice_ble/src/ble-mac.h'
```

File: dist/tools/cppcheck/check.sh [hunk 1]
```diff
@@ -53,6 +53,7 @@ if [ -z "${FILES}" ]; then
     exit
 fi
 
-cppcheck --std=c99 --enable=style --force --error-exitcode=2 --quiet -j 8 \
+# TODO: switch back to 8 jobs when/if cppcheck issue is resolved
+cppcheck --std=c99 --enable=style --force --error-exitcode=2 --quiet -j 1 \
```
Comment (miri64) [inline dist/tools/cppcheck/check.sh]: Fix provided in #6963 ;-)

**Pull request merged**